mockingServer/modules/Authentication.py

```diff
@@ -19,7 +19,9 @@ def post(self):
         return self.session_id
 
     def delete(self):
-        return make_response('Success', 200)
+        if not self.check_session_id():
+            return make_response('Logout failed', 401)
```

Comment:
Could this happen?

Reply:
Yes. We also need this, because if we don't check against the sessionID, the logout would always succeed. But the VCSA-Exporter always sends it's session ID to the server during logout. With this case we are able to fully cover the code in the logout method of the Connection class.